### PR TITLE
[Zero Trust] Update macOS Firewall device posture

### PR DESCRIPTION
**Pull request by schultetwin1**
Explain the two different firewalls on macOS and how the Zero Trust
client works with both.

### CODE REVIEW

**Review by kkrum — APPROVED** [review]
Looks great, thanks!

File: content/cloudflare-one/identity/devices/firewall.md [hunk 1]
```diff
@@ -39,9 +39,18 @@ Operating systems determine Firewall configuration in various ways. Follow the s
 
 ### On macOS
 
+macOS has two firewalls. The application based firewall and the port based firewall. The Zero Trust client will report a firewall is enabled if either firewall is running.
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/firewall.md]:
```suggestion
macOS has two firewalls: an application-based firewall and a port-based firewall. The Zero Trust client will report a firewall is enabled if either firewall is running.
```

File: content/cloudflare-one/identity/devices/firewall.md [hunk 2]
```diff
@@ -39,9 +39,18 @@ Operating systems determine Firewall configuration in various ways. Follow the s
...
+
+#### Application based firewall
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/firewall.md]:
```suggestion
#### Application-based firewall
```

File: content/cloudflare-one/identity/devices/firewall.md [hunk 3]
```diff
@@ -39,9 +39,18 @@ Operating systems determine Firewall configuration in various ways. Follow the s
...
+
 1.  Open **System Preferences** and navigate to **Security & Privacy**.
 1.  Verify that **Firewall** is **On**.
 
+#### Port based firewall
```
Comment (ranbel) [inline content/cloudflare-one/identity/devices/firewall.md]:
```suggestion
#### Port-based firewall
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**